packages/fuels/tests/bindings.rs

```diff
@@ -14,7 +14,7 @@ pub fn null_contract_id() -> Bech32ContractId {
 }
 
 #[tokio::test]
-async fn compile_bindings_from_contract_file() {
+async fn compile_bindings_from_contract_file() -> Result<()> {
```

Comment:
This test does not have any `unwraps` nor returns results so it does not benefit from adding a `Result` return type.

```suggestion
async fn compile_bindings_from_contract_file() {
```

Reply:
```rust
fn wallet_initialization_code(maybe_command: Option<InitializeWalletCommand>) -> TokenStream {
.
.
.
    quote! {
        let [#(#wallet_names),*]: [_; #num_wallets] = launch_custom_provider_and_get_wallets(
            WalletsConfig::new(Some(#num_wallets as u64), None, None),
            None,
            None,
        )
        .await?
        .try_into()
        .expect("Should have the exact number of wallets");
    }
}
```
"Since I added `.await?` to the `wallet_initialization_code`, the test now needs to return `Result` to handle the potential errors."

Reply:
We avoided passing the errors from the macro. This forces everybody that uses this macro to return a result. Instead you should `expect` with a nice message.

Reply:
done